gene2phenotype_project/gene2phenotype_app/serializers/locus_genotype_disease.py

```diff
@@ -608,18 +611,24 @@ def update(self, instance, validated_data):
             raise serializers.ValidationError(
                 {"error": f"G2P record '{instance.stable_id.stable_id}' already has confidence value {confidence}"}
             )
+        
 
         # Update confidence
+        old_confidence = instance.confidence
         instance.confidence = confidence_obj
 
         # Update the 'date_review'
         instance.date_review = get_date_now()
 
         # Save all updates
         instance.save()
+        user_obj = User.objects.get(email=user)
+        if settings.SEND_MAILS == "True":
```

Comment:
The `settings.SEND_MAILS` should be a boolean not a string.
If in the settings you use `config.getboolean('email', 'send_to_mailing_list')`, then here you can check the boolean properly.